desktop/get-started.md

```diff
@@ -3,33 +3,28 @@ description: Docker Dashboard
 keywords: Docker Dashboard, manage, containers, gui, dashboard, images, user manual
 title: Sign in and get started
 redirect_from:
-- /docker-for-windows/
-- /docker-for-windows/index/
-- /docker-for-windows/started/
-- /engine/installation/windows/
-- /installation/windows/
-- /win/
-- /windows/
-- /windows/started/
-- /winkit/
-- /winkit/getting-started/
+- /desktop/linux/
 - /desktop/linux/index/
```

Comment:
As mentioned in the top description; slightly wondering if this is the correct place for the redirect ("get started") or if these should've been to the general Docker Desktop "overview" page